src/glossary.rst

```diff
@@ -807,3 +807,63 @@ Glossary
 
       When a Cylc detects that a suite has stalled an email will be sent to the
       user. Human interaction is required to escape a stalled state.
+
+   suicide trigger
+      Suicide triggers remove :term:`tasks <task>` from the :term:`graph`.
+
+      This allows Cylc to dynamically alter the graph based on events in the
+      workflow.
+
+      .. warning::
+
+         Since Cylc 8 suicide triggers have been surpassed by
+         :term:`graph branching` which provides a simpler, superiour
+         solution.
+
+      Suicide triggers are denoted using an exclamation mark, ``!foo`` would
+      mean "remove the task foo from this cycle".
+
+      .. code-block:: cylc-graph
+
+         a => b
+
+         # suicide trigger which removes the task "b" if "a" fails
+         # NOTE: since Cylc 8 this suicide trigger is not necessary
```

Comment:
Is this line necessary in addition to the warning box above?

Reply:
This line is supposed to convey that the suicide trigger no longer has any impact on this workflow.